playbooks/roles/ocp-config/templates/chrony-machineconfig.j2

```diff
@@ -0,0 +1,25 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: {{item}}
+  name: 99-{{item}}-chrony-configuration
+spec:
+  config:
+    ignition:
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 2.2.0
+    networkd: {}
+    passwd: {}
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,{{ chronybase64.content }}
+          verification: {}
+        filesystem: root
+        mode: 420
+        path: /etc/chrony.conf
+  osImageURL: ""
```

Comment:
Please fix this with a new line.

Reply:
You might want to remove the spaces at the last line.

Reply:
You're right, sorry for that